translations/OWNERS

```diff
@@ -3,4 +3,5 @@
 reviewers:
   - brendandburns
 approvers:
+  - sig-architecture-approvers
```

Comment:
I think this is only used by kubectl, so maybe sig-cli instead?

Reply:
Done